docs/guide/commands/files.md

```diff
@@ -1,33 +1,58 @@
-# The Basics
+# Creating Commands
 
-Once you have your command files loaded, you can start writing them.
+`Commands` are files containing a `command` function.  They must be prefixed with `export default` in order to be loaded by `Jelly`.
 
-## Setup
+When a `command` is invoked, the command's `run` function is called.  This is where your custom command logic lives.
+
+## `Run`
+
+The `run` function gives you access to the [`interaction`](docs/discord.js/main/class/CommandInteractionOptionResolver) object and the [`client`](/api/client) running it.
+
+```js
+export default command({
+    run: ({ interaction, client }) => {
+        // Do something
+    }
+})
+```
+
+### interaction [`ChatInputCommandInteraction`](https://discord.js.org/#/docs/discord.js/main/class/CommandInteractionOptionResolver)
+
+The chat input command interaction data.
+
+### client [`JellyCommands`](/api/client)
+
+The client used by the command.
+
+## Command Types
+
+The 3 types of commands are `command` (for slash commands), `messageCommand`, and `userCommand`.
+
+See [`core options`](api/commands.html#core-options) for all of the ways you can configure your commands.
 
-All command files use a helper function to setup the command and then export that:
 
 ### Slash Commands
 
-Slash commands use the `command` helper, for example:
+Slash commands use the `command` helper.  Unlike the others, they accept `description` and `descriptionLocalizations` options.
```

Comment:
There are a few other bits they can have like autocomplete, there is [this page](https://jellycommands.dev/guide/commands/slash.html) that covers part of it